library/alloc/src/alloc.rs

```diff
@@ -31,8 +31,14 @@ unsafe extern "Rust" {
     #[rustc_std_internal_symbol]
     fn __rust_alloc_zeroed(size: usize, align: usize) -> *mut u8;
 
+    #[rustc_nounwind]
+    #[rustc_std_internal_symbol]
+    #[cfg(not(bootstrap))]
+    fn __rust_no_alloc_shim_is_unstable();
```

Comment:
At the very least this symbol name will need to be changed to prevent UB when someone uses the old definition.

Reply:
Good point, I'll rename it.